### ISSUE TITLE
Non-ASCII image file with layout='fill' returns 404 after build

### ISSUE THREAD

**Issue by Artoria-0x04** [vercel/next.js#20811]
**What example does this report relate to?**

none

**What version of Next.js are you using?**

10.0.4

**What version of Node.js are you using?**

14.5.0

**What browser are you using?**

Chrome 87

**What operating system are you using?**

macOS 10.15.7

**How are you deploying your application?**

local production build

**Describe the Bug**

I have some images in `/public` such as `/public/assets/services/hyjs/谭芳.png` which will be URL encoded as `/assets/services/hyjs/%E8%B0%AD%E8%8A%B3.png` when visited directly.

When I put this image into `<Image>` component like:

```typescript
<Image
  src={'/assets/services/hyjs/谭芳.png'}
  alt="lawyer"
  layout="fill"
  objectFit="cover"
  objectPosition={'top center'}
/>
```

next will request URL `/_next/image?url=%2Fassets%2Fservices%2Fhyjs%2F%E8%B0%AD%E8%8A%B3.png&w=828&q=75` and returns 404. It seems this method does not support encoded url yet.

**Expected Behavior**

Returns optimized image as requested.

**To Reproduce**

1. Put an image with non-ASCII name under `/public`.
2. Reference the image with 'fill' layout.

**Comment by Artoria-0x04** [comment]
Sorry for the unrelated label [type:example].

**Comment by styfle** [comment]
This was fixed in PR #30287 and is available with the latest version of Next.js 

Please try `npm i next@latest` or `yarn add next@latest`, thanks!
